### PR TITLE
Adding the rst file to render IO under Fluid in paddlepaddle.org

### PR DESCRIPTION
**Pull request by kavyasrinet**
This PR solves the issue : #7335 

As of now, I have added comments in just the `is_parameter` function of `io.py` and made sure this works and renders fine, and added the required rst file.
Once merged, I can add comments to other functions of the file as well in order for them to render.

### CODE REVIEW

File: python/paddle/v2/fluid/io.py [hunk 1]
```diff
@@ -11,6 +11,16 @@
...
 def is_parameter(var):
+    """Check whether the variable is a Parameter
+
+    This function checks whether the input variable is a Parameter.
+
+    Args:
+        var : The input variable.
+
+    Returns:
+        boolean result whether the variable is a Parameter
```
Comment (luotao1) [inline python/paddle/v2/fluid/io.py]: Is line 14 and line 22 lacking periods in the end?

File: python/paddle/v2/fluid/io.py [hunk 2]
```diff
@@ -11,6 +11,16 @@
 
 
 def is_parameter(var):
```
Comment (luotao1) [inline python/paddle/v2/fluid/io.py]: As `is_parameter` is not in ` __all__ `, could it show correctly on the website?

File: python/paddle/v2/fluid/io.py (hunk 1 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/io.py]: Thank you! I will fix these things. Just wanted to check if it renders the comment at all.

**Comment by kavyasrinet** [comment]
Here is a screesnhot after adding the rst:
screen shot 2018-01-11 at 7 22 56 pm

**Comment by luotao1** [comment]
From the screenshot, there is no heading `isParameter` here.

**Comment by kavyasrinet** [comment]
Ah right, that was a stale screenshot before I renamed it, here is the updated one:
screen shot 2018-01-11 at 7 50 57 pm

File: python/paddle/v2/fluid/io.py (hunk 1 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/io.py]: Fixed.

File: python/paddle/v2/fluid/io.py (hunk 2 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/io.py]: Yes, see screenshot below

File: doc/api/v2/fluid/io.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+===========
+IO
+===========
+
+
+
+isParameter
```
[the remark this answers is not in the document]
Comment (kavyasrinet) [reply doc/api/v2/fluid/io.rst]: Fixed.

**Pull request merged**